src/main/java/org/broadinstitute/hellbender/tools/gvs/common/SchemaUtils.java

```diff
@@ -83,6 +83,17 @@ public static long encodeLocation(String chrom, int position) {
         return (long) chromosomeIndex * chromAdjustment + (long) position;
     }
 
+
+    public static long encodeCompressedRefBlock(String chrom, long position, long length, long stateAsInt) {
+        long chromosomeIndex = ChromosomeEnum.valueOfContig(chrom).index;
+        long packedBytes = ((0xFFFF & chromosomeIndex) << 48) |
```

Comment:
OCD: If the high bit of `chromosomeIndex` was set (not that it theoretically could be) wouldn't this overflow a signed `long`? Maybe a `0x7FFF` mask?

Reply:
Not an incorrect observation.  But when choosing the number of bits to allocate, I did some googling.  Apparently the highest number of chromosomes ever discovered in an organism (according to Guinness World Record) is some random single-celled thing that has 15.6k.  The number of allocated bits allows for over 65k.  We have a LARGE buffer before we'd ever encounter any issues with any undiscovered organisms that could come close to overflowing the allocated bits.  It's a theoretical possibility that is impossible to actually encounter